sdk/tables/test-resources.json

```diff
@@ -74,7 +74,7 @@
                 },
                 "locations": [
                     {
-                        "locationName": "[variables('location')]",
+                        "locationName": "[parameters('cosmosdblocation')]",
```

Comment:
Where is `cosmosdblocation` defined? It doesn't seem to exist in New-TestResources.ps1